pynanoleaf/pynanoleaf.py

```diff
@@ -32,6 +34,16 @@ def _dictitem_property(keystr):
         fset=_deepsetter(key)
     )
 
+def _parseEffect(effect):
+    """These effect names have special meaning and are not directly setable
+    and should not be returned as effect name for Consumers."""
+    RESERVED_EFFECT_NAMES = ["*Solid*", "*Static*", "*Dynamic*"]
```

Comment:
I was thinking of putting this at the top (like a global variable?), would that be better, what do you think?

Reply:
it's also short enough to write inline imho

Reply:
ok